xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -992,11 +1004,33 @@ bool CXBMCApp::StartActivity(const std::string &package, const std::string &inte
     newIntent.setDataAndType(jniURI, dataType);
   }
 
+  if (!flags.empty())
+  {
+    try
+    {
+      int intentFlags = std::stoi(flags);
+      newIntent.setFlags(intentFlags);
```

Comment:
I would not spend an extra var for this. Why not just `newIntent.setFlags(std::stoi(flags));`? If you would like to keep the var, make it const, please (`const int ...`).